### PR TITLE
we wrongly accepted project as a timetracking subject

### PR DESCRIPTION
**Pull request by stivni**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -442,6 +442,10 @@ We list all backwards-incompatible changes here. As described above, new additio
   - `tasks.list`
   - `tasks.create`
   - `tasks.update`
+  
+- We won't accept `project` as a `subject` anymore in `timeTracking.add`, `timeTracking.update`, `timers.start` and `timers.update`.
+  We wrongly accepted this in the past, but time tracking always needs to be coupled to a milestone. Please use the `milestone` subject instead.
+  For older versions, we will link the time tracked to the oldest `open` milestone, if there is no `open` milestone, it's linked to the most recent `closed` or `done` milestone.
```
Comment (stivni) [inline apiary.apib]:
```suggestion
  For older versions, we will link the time tracked to the oldest `open` milestone. If there is no `open` milestone, it's linked to the most recent `closed` or `done` milestone.
```

File: src/changes-backwards-incompatible.apib [hunk 2]
```diff
@@ -16,6 +16,10 @@ We list all backwards-incompatible changes here. As described above, new additio
   - `tasks.list`
   - `tasks.create`
   - `tasks.update`
+  
+- We won't accept `project` as a `subject` anymore in `timeTracking.add`, `timeTracking.update`, `timers.start` and `timers.update`.
+  We wrongly accepted this in the past, but time tracking always needs to be coupled to a milestone. Please use the `milestone` subject instead.
+  For older versions, we will link the time tracked to the oldest `open` milestone, if there is no `open` milestone, it's linked to the most recent `closed` or `done` milestone.
```
Comment (stivni) [inline src/changes-backwards-incompatible.apib]:
```suggestion
  For older versions, we will link the time tracked to the oldest `open` milestone. If there is no `open` milestone, it's linked to the most recent `closed` or `done` milestone.
```

**Review by psrpinto — APPROVED** [review]

**Review by tiagobastos — APPROVED** [review]

**Review by kevinwaelkens — APPROVED** [review]

**Review by mikeverf — APPROVED** [review]

File: src/changes-backwards-incompatible.apib [hunk 3]
```diff
@@ -16,6 +16,10 @@ We list all backwards-incompatible changes here. As described above, new additio
   - `tasks.list`
   - `tasks.create`
   - `tasks.update`
+  
+- We won't accept `project` as a `subject` anymore in `timeTracking.add`, `timeTracking.update`, `timers.start` and `timers.update`.
```
Comment (dietrichm) [inline src/changes-backwards-incompatible.apib]: PP:
```suggestion
- We won't accept `project` as a `subject` anymore in `timeTracking.add`, `timeTracking.update`, `timers.start` and `timers.update`.  
```

File: src/changes-backwards-incompatible.apib [hunk 4]
```diff
@@ -16,6 +16,10 @@ We list all backwards-incompatible changes here. As described above, new additio
...
+  We wrongly accepted this in the past, but time tracking always needs to be coupled to a milestone. Please use the `milestone` subject instead.
```
Comment (dietrichm) [inline src/changes-backwards-incompatible.apib]: PP:
```suggestion
  We wrongly accepted this in the past, but time tracking always needs to be coupled to a milestone. Please use the `milestone` subject instead.  
```

**Review by dietrichm — APPROVED** [review]
This adds the right line breaks.

File: apiary.apib [hunk 5]
```diff
@@ -442,6 +442,10 @@ We list all backwards-incompatible changes here. As described above, new additio
   - `tasks.list`
   - `tasks.create`
   - `tasks.update`
+  
+- We won't accept `project` as a `subject` anymore in `timeTracking.add`, `timeTracking.update`, `timers.start` and `timers.update`.
```
Comment (jenssegers) [inline apiary.apib]:
```suggestion
- We have removed `project` as a `subject` in `timeTracking.add`, `timeTracking.update`, `timers.start` and `timers.update` endpoints.
```

Comment (stivni) [reply apiary.apib]:
```suggestion
- We have removed `project` as a possible `subject` in `timeTracking.add`, `timeTracking.update`, `timers.start` and `timers.update`.
```

**Pull request merged**